### PR TITLE
chore: fix lint errors, add linting to ci

### PR DESCRIPTION
**Pull request by montezume**
#### Summary

Fixes the CSS lint errors, and updates the CI config to run the linters.

### CODE REVIEW

File: src/components/grid/grid.js [hunk 1]
```diff
@@ -2,13 +2,13 @@ import PropTypes from 'prop-types';
 import styled from '@emotion/styled';
 
 const GridItem = styled.div(props => ({
+  gridArea: props.gridArea,
+  gridColumn: props.gridColumn,
```
Comment (montezume) [inline src/components/grid/grid.js]: these fixes are all about moving around the properties.

File: src/components/inputs/checkbox-input/checkbox-input.js [hunk 2]
```diff
@@ -14,17 +14,24 @@ import vars from '../../../../materials/custom-properties';
...
+  }
+  return css``;
+};
+
 const Label = styled.label`
   display: flex;
   align-items: center;
   cursor: ${props => (props.disabled ? 'not-allowed' : 'pointer')};
   position: relative;
-  ${props =>
```
Comment (montezume) [inline src/components/inputs/checkbox-input/checkbox-input.js]: linting doensn't work with these type of dynamic styles, but I think the other ones are more readable anyways.

File: src/components/inputs/toggle-input/toggle-input.js [hunk 3]
```diff
@@ -61,29 +66,29 @@ const Span = styled.span`
...
     background: ${vars.colorNeutral};
   }
 
-  &:disabled&:checked + ${Span}:before {
+  &:disabled&:checked + ${Span}::before {
     background: ${vars.colorPrimary25};
   }
 
-  &:checked + ${Span}:after {
+  &:checked + ${Span}::after {
```
Comment (montezume) [inline src/components/inputs/toggle-input/toggle-input.js]: should use `::after` and not `:after`

Comment (tdeekens) [reply src/components/inputs/toggle-input/toggle-input.js]: Ah nice. I always wondered if there was a preferred way.

**Comment by tdeekens** [comment]
This is where VRTs are just the best thing on earth.

**Comment by montezume** [comment]
@tdeekens yeah, I had mixed up one of the conversions and ended up adding padding if false instead of if true, and VRTs caught it 😄

**Pull request merged**